### DISCUSSION TITLE
Playing multiple WebRTC streams on a single WebSocket.

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5540] [FAQs]
Hi,
Is it possible to play multiple WebRTC streams on a single websocket connection?
If yes, is there an example that I can refer?

**Comment by Mohit-3196** [comment]
Thank you for your great question.

Yes, Ant Media Server supports playing multiple WebRTC streams on a single websocket connection.
Here's a sample code that can be referred to see how how to play 2 different WebRTC streams on a single websocket 
https://codepen.io/Burak-Kekec/pen/mdQveNj

Thank you
